### PR TITLE
add e2e functional test steps to hack/README.md

### LINKED ISSUE #327: Update and add new make functest instructions to hack/README.md
after PR #314 - need to also add updates to the hack/README.md with basic description and instructions on how to run this

### PR DESCRIPTION
**Pull request by screeley44**
fixes #327 

Add simple instructions on how to execute the `make functest` properly

### CODE REVIEW

**Comment by awels** [comment]
lgtm

File: hack/README.md [hunk 1]
```diff
@@ -69,6 +70,21 @@ These may be passed to a target as `$ make VARIABLE=value target`
 - `VERBOSITY`: (default: 1) Set global log level verbosity
 - `PULL_POLICY`: (default: IfNotPresent) Set global CDI pull policy
 
+#### Execute Functional Tests
+To Run Tests
+```
```
Comment (aglitke) [inline hack/README.md]: We need to document the KUBEVIRT_PROVIDER variable and add a link to the available options.

Maybe https://github.com/kubevirt/kubevirtci/blob/master/README.md ?

Comment (awels) [reply hack/README.md]: Well we specifically opted to support just k8s-1.10.4 and os-3.10.0 right now, to keep the logic that determines which image to grab simple, with the provision that if k8s-1.11.x or os-3.11.x comes out that we will switch to those. So those two are the only actual options available right now.

File: hack/README.md [hunk 2]
```diff
@@ -69,6 +70,21 @@ These may be passed to a target as `$ make VARIABLE=value target`
...
+ make cluster-up
```
Comment (jeffvance) [inline hack/README.md]: My version of this command was `KUBEVIRT_PROVIDER=k8s-1.10.4 KUBEVIRT_NUM_NODES=3 make cluster-up`. Are these variables not needed? If so what's their default?

File: hack/README.md [hunk 3]
```diff
@@ -69,6 +70,21 @@ These may be passed to a target as `$ make VARIABLE=value target`
...
+ make cluster-sync
```
Comment (jeffvance) [inline hack/README.md]: I'd like to see a short description of each of these targets.

File: hack/README.md (hunk 2 above)
Comment (awels) [reply hack/README.md]: Technically they are not needed, as their defaults are k8s-1.10.4 and the number of nodes = 1. The environment variables allow you to override the defaults with something else like increase the number of nodes or use the open shift provider.

Comment (jeffvance) [reply hack/README.md]: @awels I didn't realize there was a default for `KUBEVIRT_PROVIDER`. Good to know and I'll close issue #331

File: hack/README.md [hunk 4]
```diff
@@ -69,6 +70,28 @@ These may be passed to a target as `$ make VARIABLE=value target`
 - `VERBOSITY`: (default: 1) Set global log level verbosity
 - `PULL_POLICY`: (default: IfNotPresent) Set global CDI pull policy
 
+#### Execute Functional Tests
+Environment Variables and Supported Values
```
Comment (jeffvance) [inline hack/README.md]: +1 nice!

**Comment by jeffvance** [comment]
lgtm

**Pull request merged**